cmake/FindCVC5.cmake

```diff
@@ -0,0 +1,33 @@
+if (USE_CVC5)
+    find_path(CVC5_INCLUDE_DIR cvc5/cvc5.h)
+    find_library(CVC5_LIBRARY NAMES cvc5)
+    include(FindPackageHandleStandardArgs)
+    find_package_handle_standard_args(CVC5 DEFAULT_MSG CVC5_LIBRARY CVC5_INCLUDE_DIR)
+    if(CVC5_FOUND)
+        # CVC5 may depend on either CLN or GMP.
```

Comment:
Is this still true? It would actually simplify some stuff if we could ignore CLN

Reply:
CLN is now listed among optional dependencies: https://cvc5.github.io/docs/cvc5-1.0.2/installation/installation.html#cln-v1-3-class-library-for-numbers